### PR TITLE
Code style and README tweaks

### PR DESCRIPTION
**Pull request by joankaradimov**
I've shortened the code using some Groovy syntax.
I've also removed the explicit `flexmark-java` version from the readme.

On an unrelated note: is there any chance for a new release soon?

### CODE REVIEW

File: src/main/groovy/de/fntsoftware/gradle/MarkdownToPdf.groovy [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+	private static final DEFAULT_RESOURCE = 'bitbucket.css'
 
-	String cssFile;
+	String cssFile
 
-	private String cssFileContent = null;
+	private String cssFileContent = null
 
 	public String getCssFileContent() {
 		if(this.cssFileContent == null) {
```
Comment (adanielsmeyer) [inline src/main/groovy/de/fntsoftware/gradle/MarkdownToPdf.groovy]: Can you please also add a whitespace here.

**Comment by adanielsmeyer** [comment]
Thanks for the changes you already made. Maybe i can publish the new version next week with your changes.

File: src/main/groovy/de/fntsoftware/gradle/MarkdownToPdf.groovy (hunk 1 above)
Comment (joankaradimov) [reply src/main/groovy/de/fntsoftware/gradle/MarkdownToPdf.groovy]: I missed that one, yes. I'll fix it.

**Comment by joankaradimov** [comment]
I've force pushed a fix for the missing whitespace.

> Maybe i can publish the new version next week with your changes.

Thanks! Looking forward to the release.

**Pull request merged**